bindings/python/countdown-n-lap.py

```diff
@@ -0,0 +1,81 @@
+'''
```

Comment:
Hello @vsumpi, if you look at the source tree, you'll see that there are already a bunch of examples written in Python, but they are in the `bindings/python/samples` directory, rather than in `bindings/python`

Reply:
Okay, I have a question, how to set flags globally on my rpi so it is not embedded in the file, is it possible?